receiver/webhookeventreceiver/README.md

```diff
@@ -64,6 +64,21 @@ Three log records will be created from this example. The first two are JSON body
 
 This receiver does not attempt to marshal the body into a structured format as it is received so it cannot make a more intelligent determination about where the split records. 
 
+### Split logs at JSON boundary example
+
+If you don't have clear new-line splits between objects, the regex `}\s.{` will match the boundary between objects.
```

Comment:
``}\s.{` - but you can have two objects with no separators, such as `{"foo":"bar"}{{"foo":"foobar"}` how do you handle this?

Reply:
actually you use `}\s*{` in the code